src/app/register-email-form/register-email-form.component.ts

```diff
@@ -66,11 +66,11 @@ export class RegisterEmailFormComponent implements OnDestroy, OnInit {
 
   subscriptions: Subscription[] = [];
 
-  captchaVersion(): Observable<string> {
+captchaVersion(): Observable<string> {
     return this.googleRecaptchaService.captchaVersion();
   }
 
-  captchaMode(): Observable<string> {
+captchaMode(): Observable<string> {
```

Comment:
Please remove the changes from this file, as they appear to be unnecessary.  You've just changed the alignment of the methods here...and that isn't needed.